torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -191,6 +191,17 @@ bool needsProfiledOutput(Node* n) {
   }
 }
 
+static void removeProfileCounter(Block* b) {
+  for (auto it = b->nodes().rbegin(); it != b->nodes().rend();) {
+    auto n = *it;
+    if (n->kind() == prim::profile && n->inputs().size() == 0) {
+      it.destroyCurrent();
```

Comment:
Should we return right after? There couldn't be more than 1 such node, right?

Reply:
yup, i actually had this exact same thought, but i guess attending to a few things didn't work out too well(((((